src/html_support_files/dune

```
@@ -2,7 +2,7 @@
 ; of the vendored projects.
 
 (rule
- (alias runtest)
+ (alias html-support-files)
```

Can it be both (we want the tests to detect an unpromoted html_support_files)?

---

Nit-picking, but why so long?
```suggestion
 (alias support-files)
```

---

Dune's [manual](https://dune.readthedocs.io/en/stable/dune-files.html#rule) suggests using `(aliases runtest support-files)`.